test/e2e/framework/framework.go

```diff
@@ -90,3 +91,8 @@ func (f *Framework) DeleteServiceExport(cluster framework.ClusterIndex, name str
 	By("Waiting for mcs cleanup...")
 	time.Sleep(cleanupWait)
 }
+
+func (f *Framework) GetService(cluster framework.ClusterIndex, name string, namespace string) (*v1.Service, error) {
```

Comment:
should we use wait and retry in case service is not yet created?

Reply:
Or in case it globalIP is not set, but we know that we are testing global ip?

Reply:
That can be written by wrapping this around an AwaitUntil if needed. In this particular case we know that we've created the service. This will get replaced with AwaitUntilGlobalIp method once I have the globalnetflag check.